src/main/java/org/gridsuite/modification/server/ModificationType.java

```diff
@@ -41,7 +41,8 @@ public enum ModificationType {
     BRANCH_STATUS_MODIFICATION(PreloadingStrategy.NONE),
     DELETE_VOLTAGE_LEVEL_ON_LINE(PreloadingStrategy.NONE),
     DELETE_ATTACHING_LINE(PreloadingStrategy.NONE),
-    GENERATION_DISPATCH(PreloadingStrategy.COLLECTION);
+    GENERATION_DISPATCH(PreloadingStrategy.COLLECTION),
+    MASSIVE_EQUIPMENTS_MODIFICATIONS(PreloadingStrategy.COLLECTION);
```

Comment:
Don't call it massive this is not their right term in English. -> maybe TABLE_EQUIPMENT_MODIF

Reply:
Renamed